### ISSUE TITLE
Deck de slides para la charla de avance

### ISSUE THREAD

**Issue by lgmoyano** [martinfama/LPGNN#3]
Esto lo planteamos en la última reunión. Sería bueno tener para el viernes una versión draft, aunque sea planteando contenido via títulos nomás, pero que haya un esqueleto lo más completo posible. Apunta a 20-25 slides nomás. Cosas que tiene que tener:

- [ ] Una intro general al tema
- [ ] Un racconto muy breve de lo hecho en el primer semestre
- [ ] Alguna slide que exponga el trabajo a partir de ahora, que debe tener cierta lógica de conexión con lo anterior
- [ ] Alguna explicación básica de las nuevas técnicas (GNNs)
- [ ] Trabajo hecho en este semestre
- [ ] Resultados preliminares
- [ ] Camino futuro

**Comment by martinfama** [comment]
Hoy me estoy dedicando a armar los slides a partir de lo nuevo. 

1.  Intro a GNN's. Lo esencial de la estructura de estas redes, como la estructura de grafo computacional dada por la misma red. La distinción entre los atributos de nodos y estructura de la red en cuanto a el rol correspondiente en la computación de la GNN. 
2. GNN's y el laburo del año pasado. Entender el proceso que hay entre tener una red (como por ejemplo una PS,) y obtener una lista de scores para predicción de enlaces. Osea, preparar los atributos de nodos, train/val/test, la GNN (comentar que hay distintos modelos) y que es lo que calcula la GNN. Embeddings, y la posibilidad de encarar por el lado de ir a espacio hiperbólico con la red. Tengo que entender mejor como modificar la estructura de los datos que saca la GNN (mas alla de simplemente cambiar la dimensionalidad). ¿Con la función de pérdida solamente? Estas cosas me doy cuenta que debería tener más afiladas, poder modelar mejor la salida de la red. Por ahora es la loss es binary_cross_entropy_with_logits, lo cuál es una función de pérdida decente para nuestro caso, pero el output que la loss usa de la red es simplemente el resultado de una GNN como las describen en un paper. 
3. Bueno acá entrarían los datos. Voy a armar un mejor catálogo de datos. Para algo ya está el código bastante más organizado, así que voy a hacer un poco más de barrido. Voy a tener para poner curvas PR para distintos parámetros. Vamos a ver que por ahora LaBNE y ponele Common Neighbours son vastamente superiores a los resultados que estamos teniendo. También voy a armar más resultados con lo del attribute passing, que ahora que lo estuve viendo quizás no es tan "gradual" el cambio como pensaba. Mostrar la salida de la GNN como un embedding.
